jax/_src/nn/functions.py

```diff
@@ -121,8 +122,7 @@ def squareplus(x: ArrayLike, b: ArrayLike = 4) -> Array:
     x : input array
     b : smoothness parameter
   """
-  numpy_util.check_arraylike("squareplus", x)
-  numpy_util.check_arraylike("squareplus", b)
+  numpy_util.check_arraylike("squareplus", x, b, emit_warning=True)
```

Comment:
Since this function checked the validity of inputs previously, we should not set `emit_warning=True`.